client/cmd_selected_test.go

```diff
@@ -499,8 +499,8 @@ func TestClient_Copy_Uid(t *testing.T) {
 	}()
 
 	tag, cmd := s.ScanCmd()
-	if cmd != "UID COPY 78:102 Drafts" {
-		t.Fatalf("client sent command %v, want %v", cmd, "UID COPY 78:102 Drafts")
+	if cmd != "UID COPY 78:102 \"Drafts\"" {
```

Comment:
```
uid             = "UID" SP (copy / fetch / search / store)
                    ; Unique identifiers used instead of message
                    ; sequence numbers
copy            = "COPY" SP sequence-set SP mailbox
```